src/factories/postburn/PostBurnMCParticles_factory.h

```diff
@@ -0,0 +1,53 @@
+// Copyright 2024, Alex Jentsch, Jihee Kim, Brian Page
+// Subject to the terms in the LICENSE file found in the top-level directory.
```

Comment:
Use SPDX copyright header for all new files.

Reply:
Done.